crates/sui-indexer-alt-framework/src/handlers/cp_sequence_numbers.rs

```diff
@@ -22,6 +23,80 @@ pub struct StoredCpSequenceNumbers {
 
 pub struct CpSequenceNumbers;
 
+/// A struct that can be instantiated by the pruner task to map a `from` and `to` checkpoint to its
```

Comment:
I think we've graduated to the point where this and `StoredCpSequenceNumbers` deserves its own module -- maybe you can move `watermarks` in there too, so we have a `model/{watermarks.rs, cp_sequence_numbers.rs}`?

Reply:
sure thing, i'll move the `watermarks` in a separate PR